### DISCUSSION TITLE
Vulnerability issue: Any update on the new version

### DISCUSSION THREAD

**Discussion by artkamote** [AutoFixture/AutoFixture#1319] [Q&A] [answered]
Getting vulnerability issue CVE-2019-0820 on v 4.17

![image](https://user-images.githubusercontent.com/85152202/150536109-f4762621-40b1-4df7-9c38-d98af1e1f569.png)

**Comment by aivascu** [comment]
@artkamote thank you for raising this. I'll try to release a patch version ASAP.

**Comment by aivascu** [comment]
@artkamote have you tried installing a higher version of `System.Text.RegularExpressions` in your test projects?

**Reply by artkamote** [reply]
Hi @aivascu ,

I haven't installed it directly.

**Reply by aivascu** [reply]
@artkamote could you try to install it? Theoretically if you install the package into your test projects, Fare and DataAnnotations should use your local package, instead of the minimum required version.

**Reply by artkamote** [reply]
It did work except for conflict with System.Text package. ☹️ 
Thanks.

**Comment by jnyrup** [comment]
[Fare 2.2.0](https://www.nuget.org/packages/Fare/2.2.0) has been released with a netstandard2.1 target, which shouldn't trigger nuget scanners about a vulnerable `System.Text.RegularExpressions`.
As AutoFixture 4.17 doesn't have any targets that can consume a netstandard2.1 package, this won't currently change the situation.

But with AutoFixture 5 adding support for netstandard2.1 in #1331, we could bump Fare to 2.2.0 and get rid of the warning for many users of AutoFixture.

**Reply by jnyrup** [reply]
[Fare 2.2.1](https://www.nuget.org/packages/Fare/2.2.1#dependencies-body-tab) has been released with a .netstandard2.0 target.
It should now be now possible for AutoFixture 4.17 to update it's version of Fare to fix the security warnings for users consuming the .netstandard2.0 target of AutoFixture.

**Reply by Marty1703** [reply]
@jnyrup, your suggestion sounds good, but why why is Autofixture still referencing Fare 2.1.1?

**Reply by aivascu** [reply]
The Fare dependency has been updated on the `release/5.0.0` branch and is available on NuGet.org after [5.0.0-preview0006](https://www.nuget.org/packages/AutoFixture/5.0.0-preview0006).

**Reply by Marty1703** [reply]
The same applies to System.ComponentModel.Annotations. 

![image](https://github.com/AutoFixture/AutoFixture/assets/14041814/fbaa4b74-b0a7-417c-8e32-4c9e162f359d)

**Reply by aivascu** [reply]
@Marty1703 might sound stupid, but what tool are you using? 😅Just so we don't go back and forth on each transitive dependency.

**Reply by Marty1703** [reply]
Dependency-Track, but also other tools, like @artkamote used returns the same tree.

**Reply by aivascu** [reply]
@Marty1703 I wasn't aware of these tools. So far I've been able to find a feature in the dotnet CLI and some enterprise solutions nowhere near my budget. Thank you!

**Comment by aivascu** [comment, marked as answer]
The Fare dependency has been updated on the release/5.0.0 branch and is available on NuGet.org after [5.0.0-preview0006](https://www.nuget.org/packages/AutoFixture/5.0.0-preview0006).
